src/Microsoft.Health.Fhir.Api/Microsoft.Health.Fhir.Api.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
We use variables for some of our package versions, like the FHIR version. Would it be possible to do the same here so that future updates would only have to change 1 file?

Reply:
I was too lazy, but if you insist :)